### DISCUSSION TITLE
【求助】大佬们，求一个便宜好用的梯子，求求了！

### DISCUSSION THREAD

**Discussion by 0build0** [share121/inter-knot#3049] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >
![1723690768708](https://github.com/user-attachments/assets/1c18f1a2-bfb2-4bf7-a41b-5efb9abb324e)
感谢大佬们！

**Comment by RaphaelHider** [comment]
很简单，分三个步骤
1.考上北京大学（本科或研究生均可）
2.在北大树洞查询“wallesspku”
3.使用校园门户账号按教程注册使用clash
免费且安全稳定<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/838dba51bd47c248d1f02b2e24a28b18_6640243918864203211.png" alt="米游兔-加油" width="75" >

**Reply by 0build0** [reply]
布什，戈门

**Reply by RaphaelHider** [reply]
咳咳，我的意思是，如果你还在校园，不妨去查询一下有没有学校提供的vpn资源，或者学长建立的vpn平台。因为学术方面经常用到科学上网。

**Reply by 0build0** [reply]
可我是高中牲🌚

**Comment by fengshanzzz** [comment]
分享传播这玩意好像是违法的，虽然用本身也违法就是了（

**Comment by lllliufeng** [comment]
不能就在git上搜？（

**Comment by UTAC99645** [comment]
我个人用猫猫云感觉不错
<a href="https://猫猫云.com">猫猫云.com</a>

**Reply by 0build0** [reply]
感谢大佬

**Reply by blackfcat** [reply]
好用吗

**Comment by QianYanYe** [comment]
小火箭也不错

**Reply by 0build0** [reply]
感谢分享

**Comment by yzysrewrwqr** [comment]
![55298923ffed0110e2c1b97e32aed95](https://github.com/user-attachments/assets/205c2ef0-b8a7-4d4e-9360-d88d95ec9fe5)
我用的这个，设计师专用的，可以包流量，避免浪费<img src="https://bbs-static.miyoushe.com/static/2024/03/21/f5f9f9902140f168d5b804fe6c365ea6_4229255832036354153.png" alt="丽都漫步-哲" width="75" >

**Reply by 0build0** [reply]
感谢感谢

**Reply by CeceliaHarex** [reply]
这个叫什么名字，谢谢大佬

**Reply by yzysrewrwqr** [reply]
> 这个叫什么名字，谢谢大佬

https://share.sj8.ca/u9359136196
看看能点开不

**Comment by pkgfedgjjtd** [comment]
TM也可以

**Comment by opqphp** [comment]
Clash  然后找一些机场（机场大部分都比专门用来翻墙的软件便宜）<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Mrguanboyuan** [comment]
就用class吧,然后找一些配置文件导入进去就行了

**Comment by baohuwofangmiaomiao** [comment]
不快，但是是免费的，可以凑合着用
分享一个免费好用的加速器, Google Play: https://play.google.com/store/apps/details?id=net.lab.flying, 官网：https://svc955.github.io

**Comment by ghost** [comment]
十块钱200个g永久的要吗不过缺点就是得经常更新失效快

**Comment by 1luik** [comment]
sdkdns

**Comment by LuminSAMA** [comment]
我用的这个，goos999.com<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Comment by muhan-972** [comment]
SakuraCat樱花猫，有按流量计费的套餐，比较适合不是经常用的：[https://dokidokiweb.com/](url)
